### DISCUSSION TITLE
how to  install tabby plugins offline

### DISCUSSION THREAD

**Discussion by LeWinCheng** [Eugeny/tabby#6149] [Q&A]
tabby version: 1.0.174
I want to install the tabby plugin, but I can't install it because it's always in the process of getting it.

![image](https://user-images.githubusercontent.com/42331885/163093999-4ff65c87-d24a-4d9a-b618-0779a842fb6c.png)

**Comment by Eugeny** [comment]
Plugins sometimes have multiple dependencies, so it's best to install it on an online machine first, and then copy over the plugins folder's contents

**Reply by LeWinCheng** [reply]
Thanks for your reply, I'll try to see if I can download the plugin in other ways。
